tensorflow/1.10/README.md

```diff
@@ -1,364 +1,154 @@
+```shell
+$ dcos package search tensorflow
+NAME             VERSION           SELECTED  FRAMEWORK  DESCRIPTION
+beta-tensorflow  0.1.0-1.3.0-beta  False     True       TensorFlow on DC/OS
+```
+This indicates that `beta-tensorflow` with a package version of `0.1.0-1.3.0-beta` is available.
 
-Using `MOUNT` volumes requires [additional configuration on each DC/OS agent system](https://docs.mesosphere.com/1.10/storage/mount-disk-resources/),
-so the service currently uses `ROOT` volumes by default. To ensure reliable and consistent performance in a production environment,
-you should configure `MOUNT` volumes on the machines that will run the service in your cluster and then configure the following as `MOUNT` volumes:
+When running:
+```shell
+$ dcos package describe beta-tensorflow
+```
+The configurable options in the DC/OS TensorFlow package and their defaults are presetnted.
 
-<a name="placement-constraints"></a>
-### Placement Constraints
+### Get to know the MNIST example
 
-Placement constraints allow you to customize where the service is deployed in the DC/OS cluster.
+One of the examples in the DC/OS TensorFlow tools repository is [MNIST](https://www.tensorflow.org/get_started/mnist/beginners) which is used as a basic example in many machine learning applications.
 
-Placement constraints support all [Marathon operators](http://mesosphere.github.io/marathon/docs/constraints.html)
-with this syntax: `field:OPERATOR[:parameter]`. For example, if the reference lists `[["hostname", "UNIQUE"]]`, use `hostname:UNIQUE`.
+When looking at the contents of the `examples/mnist.json` file that was cloned as part of the `dcos-tensorflow-tools` repo, a number of sections are noted. The ones of interest for the purpose of this introduction are `service` and `worker`.
 
-A common task is to specify a list of whitelisted systems to deploy to. To achieve this, use the following syntax for the placement constraint:
+#### Job definition
 
+The `service` section defines the properties of the job to be executed. Most importantly, `job_url` defines the location (URI) of an archive containing the job definition. Additional `job_*` properties are related to the structure of the archive defined in `job_url` as well as settings specific to the job:
+```json
+    "job_url": "https://downloads.mesosphere.com/tensorflow-dcos/examples/tf_examples-master.zip",
+    "job_path": "tf_examples-master",
+    "job_name": "demo",
+    "job_context": "{\"learning_rate\":0.5,\"num_training_steps\":1000000}",
 ```
-hostname:LIKE:10.0.0.159|10.0.1.202|10.0.3.3
+also note that the service `name` is specified as `"mnist"`.
+
+In order to better understand this, we can download and extract the `job_url` archive and check the contents:
+```shell
+$ wget -q https://downloads.mesosphere.com/tensorflow-dcos/examples/tf_examples-master.zip
+$ unzip tf_examples-master.zip
+$ ls tf_examples-master/
+README.md  benchmarks demo.py    mnist.py
```

Comment:
It might be confusing that we are using a file called `demo.py` when there is also a file called `mnist.py` in the same directory. This is my fault, since `mnist.py` is an older example that I had in the same GitHub repo that we moved over to downloads.mesosphere.com. But it might be helpful to remove the `mnist.py` file from that zip for clarity.

Reply:
I removed it from the archive and will update this section.